### ISSUE TITLE
Object detection with KITTI doesn't work right

### ISSUE THREAD

**Issue by hadi-ghnd** [NVIDIA/DIGITS#1501]
I'm  trying to perform object detection with digits based on [this](https://github.com/NVIDIA/DIGITS/tree/master/examples/object-detection) tutorial. So far I have done the instructions step by step exactly as it says. But while training the model I notice that the output chart doesn't look right and it's very different from the one on the tutorial page.

Up to epoch 5, My output chart looks like this:
[my chart](http://uupload.ir/files/ddbs_screenshot_from_2017-03-09_13-14-01.png)

While the one in the tutorial looks like this:
[the right chart](http://uupload.ir/files/ddbs_screenshot_from_2017-03-09_13-14-01.png)

Also my **KITTI** object detection dataset has different infos from the one in the tutorial page. Mine has more **Entry Counts** on the training DB and less on the validation DB. Also the **label shapes** are different.

# My created dataset:

> ## Create train_db DB
> 
> Entry Count
>     6373 
> Feature shape
>     (3, 384, 1248) 
> Label shape
>     (1, 25, 16) 

>##  Create val_db DB
> 
> Entry Count
>     1108 
> Feature shape
>     (3, 384, 1248) 
> Label shape
>     (1, 19, 16) 

# The created dataset on the [tutorial page](https://github.com/NVIDIA/DIGITS/tree/master/examples/object-detection):

> ## Create train_db DB
> 
> Entry Count
>     5984
> Feature shape
>     (3, 384, 1248) 
> Label shape
>     (1, 23, 16) 

>##  Create val_db DB
> 
> Entry Count
>     1496 
> Feature shape
>     (3, 384, 1248) 
> Label shape
>     (1, 25, 16) 

Which part have I done wrong?
Appreciate your help..

**Comment by DeepBernhard** [comment]
Hey @hadi-ghnd, dear MachineLearning lords.

I am having similar issues with the tutorial data. I had already multiple runs with 30 epochs and the best result I got was this: 
![screenshot_20170309_132540](https://cloud.githubusercontent.com/assets/26299848/23750166/f4d10968-04cb-11e7-9fa4-095006d7ed0a.png)

As for @hadi-ghnd the dataset looks different from the one in the one shown in the tutorial with 6373 entries in the train_db and 1108 in the val_db. Is it possible that the data has changes and the precision on the training and validation data has decreased with the tutorial setup?

Would be happy for some inputs, Thx
Bernhard

**Comment by gheinrich** [comment]
That looks pretty good to me. Remember to read mAP, precision and recall using the left axis (the tutorial used the scale from the right axis), admittedly this is confusing.

**Comment by DeepBernhard** [comment]
ahhh, damn, thank you so much. it had to be something i got wrong *g*

**Comment by hadi-ghnd** [comment]
Mine still doesn't look good after 8 epochs:
![screenshot from 2017-03-09 16-53-29](https://cloud.githubusercontent.com/assets/24352869/23752067/1c4a8826-04e9-11e7-87b0-4e83c305a4d0.png)

**Comment by gheinrich** [comment]
Is it better after more epochs @hadi-ghnd ? If not, you might want to verify your hyperparameters (learning rate, optimizer, etc.).

**Comment by gheinrich** [comment]
Closing due to inactivity.
